internal/policygen/schema.go

```diff
@@ -64,11 +64,6 @@ properties = {
 var orgPoliciesSchema = []byte(`
 title                = "Google Cloud Organization Policies Config Schema"
 additionalProperties = false
-required = [
-  "parent_type",
-  "parent_id",
-  "allowed_policy_member_customer_ids",
```

Comment:
do we still want `allowed_policy_member_customer_ids` to be required?

Reply:
Done